app/assets/stylesheets/checkin.scss

```diff
@@ -1,9 +1,23 @@
 @import "colors";
 
 .disable-field-set {
-  background: $gray;
+  background: #ddd;
+  color: #ddd;
 }
 
 .disable-field-set h5 {
-  color: $shadow;
+  color: #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.